spec/models/tag_spec.rb

```diff
@@ -31,4 +31,14 @@
       expect(Tag.with_active_pull_requests).to eq([ember_tag])
     end
   end
+
+  describe "#pull_requests_count" do
+    it "returns a number of active pull requests" do
+      rails_tag = create(:tag, name: "rails")
+      create(:pull_request, status: "in progress", tags: [rails_tag])
+      create(:pull_request, status: "completed", tags: [rails_tag])
+
+      expect(rails_tag.pull_requests_count).to eq(1)
```

Comment:
What do you think about moving this test to the `pull_request` model?

If you do that, you can verify which one is actually being associated with the `.active` method, and not just a count.

```ruby
active = create(:pull_request, status: "in progress")
_completed = create(:pull_request, status: "completed")

expect(PullRequest.active).to eq([active])
```